README.md

```diff
@@ -248,27 +248,23 @@ export NVM_DIR="$HOME/.nvm"
 
 ### Multiple commands
 
-By design and just like `scripts` defined in `package.json`, husky will run hook scripts as a single command. 
+To run multiple commands in series you can specify an array of commands.
 
 ```json
-"pre-commit": "cmd && cmd"
+"pre-commit": [
+  "cmd",
+  "cmd"
+]
 ```
 
-That said, if you prefer to use an array, the recommended approach is to define them in `.huskyrc.js` or `husky.config.js`.
+Or if you use a string instead, just like `scripts` defined in `package.json`, husky will run hook scripts as a single command. 
 
-```js
-const tasks = arr => arr.join(' && ')
-
-module.exports = {
-  'hooks': {
-    'pre-commit': tasks([
-      'cmd',
-      'cmd'
-    ])
-  }
-}
+```json
+"pre-commit": "cmd1 && cmd2"
 ```
 
+ℹ️ _The two examples above will result equivalent behavior._
```

Comment:
Apparently I can't write correct English sentences :sweat_smile: My apologies. Could you update this sentence to include the word "in"?

Also, to provide better support for text editors that don't fully support unicode, I suggest using `:information_source: rather than ℹ️

```suggestion
:information_source: _The two examples above will result in equivalent behavior._
```

Reply:
I don't think Github will be able to parse that 🤔

Reply:
GitHub is definitely able to parse. However, some other Markdown parsers may not be able to. 

As a demonstration, this next character is written as `:information_source:` (I didn't use GitHub's autocomplete function): :information_source:, you can try it for yourself in a new comment and hit the preview button :wink:

Reply:
Rather than continue to hold up this commit over a trivial change to the README, can we merge this PR now and update the README in a separate commit? 

I will submit a PR for the README myself if need be.

Reply:
I intentionally marked this review as a "comment" (not "changes requested") as it is just a suggestion. From my point of preview this is not preventing a merge... But I'm not in the position of merging this :upside_down_face: Whether or not the README _should_ be updated is up to the maintainers of the package